modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -5,20 +5,66 @@ This module is included in the following assemblies:
 
 :_mod-docs-content-type: PROCEDURE
 [id="ossm-configuring-distr-tracing-tempo_{context}"]
-= Configuring the distributed tracing platform (Tempo)
+= Configuring the distributed tracing platform (Tempo) and the Red Hat build of OpenTelemetry
 
-You can expose tracing data to the {TempoName} stack by appending a named element and the `zipkin` provider to the `spec.meshConfig.extensionProviders` specification in the `ServiceMehControlPlane`, as shown in the following example. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
+You can expose tracing data to the {TempoName} by appending a named element and the `opentelemetry` provider to the `spec.meshConfig.extensionProviders` specification in the `ServiceMehControlPlane`. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the OpenTelemetry Collector endpoint.
 
-You can create a TempoStack instance in the `tracing-system` namespace _after_ creating the `ServiceMeshControlPlane` and the `ServiceMeshMemberRoll` resources.
+//There is no common attribute for OpenTelemetry Collector.
+
+You can create a {OTELName} instance in a mesh namespace and configure it to send tracing data to a tracing platform backend service.
 
 .Prerequisites
 
-* You have installed the {TempoOperator} and {SMProductName} Operator in the `openshift-operators` namespace.
-* You have created namespaces such as `istio-system` and `tracing-system`.
+* You have created a TempoStack instance using the Red Hat {TempoOperator} in the `tracing-system` namespace. For more information, see "Installing the distributed tracing platform (Tempo)" in the "Additional resources" section.
+
+* You have installed the {OTELOperator} in the namespace `openshift-operators`. For more information, see "Installing the {OTELName}" in the "Additional resources" section.
 
 .Procedure
 
-. Configure the `ServiceMeshControlPlane` resource to define an extension provider:
+. Create an OpenTelemetry Collector instance in a mesh namespace such as `bookinfo`:
++
+[source, yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
```

Comment:
@yxun 
There are two warnings when I am trying this yaml with the latest operators:
```
Warning: OpenTelemetryCollector v1alpha1 is deprecated. Migrate to v1beta1.
Warning: Collector config spec.config has null objects: processors.batch:, receivers.jaeger.protocols.grpc:, receivers.jaeger.protocols.thrift_binary:, receivers.jaeger.protocols.thrift_compact:, receivers.jaeger.protocols.thrift_http:, receivers.otlp.protocols.http:, receivers.zipkin:. For compatibility with other tooling, such as kustomize and kubectl edit, it is recommended to use empty objects e.g. batch: {}.
```

Reply:
I don't understand why these unset fields are included in `spec.config`. Can these be omitted?